### ISSUE TITLE
Loss becomes nan after 1k training steps

### ISSUE THREAD

**Issue by xf-zhao** [MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells#15]
Hi, thank you for your great work!

I run this experiments according to the [wiki](https://github.com/MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells/wiki/How-to-Train-on-Your-Dataset) step by step. I did not change any hyper-parameters except that I set gpu=0 to use a cpu for training. But I found that the loss became nan just after 1K steps training.

```
...
step 1655 - loss        nan - moving ave loss        nan
step 1656 - loss        nan - moving ave loss        nan
Finish 92 epoch(es)
step 1657 - loss        nan - moving ave loss        nan
step 1658 - loss        nan - moving ave loss        nan
...
```

Do you have any idea or have you ever observed this kind of strange thing? Thank you in advance for your help!

**Comment by MahmudulAlam** [comment]
Hello @xf-zhao 

Reduce the learning rate, that would solve the issue.
